src/test/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/NotificationsPublisherTest.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.blobrouter.servicebus.notifications;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.QueueClient;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import uk.gov.hmcts.reform.blobrouter.servicebus.notifications.model.NotificationMsg;
+import uk.gov.hmcts.reform.blobrouter.util.MessageBodyRetriever;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.willThrow;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class NotificationsPublisherTest {
+
+    private NotificationsPublisher notifier;
+
+    @Mock
+    private QueueClient queueClient;
+
+    @BeforeEach
+    void setUp() {
+        notifier = new NotificationsPublisher(queueClient, new ObjectMapper());
+    }
+
+    @Test
+    void notify_should_send_message_with_right_content() throws Exception {
```

Comment:
may be instead of notify , publishing  or sth else as that method is not notifying?

Reply:
renamed tests